daemon/cmd/daemon_main.go

```diff
@@ -115,9 +115,12 @@ var (
 			}
 
 			// Open socket for using gops to get stacktraces of the agent.
-			if err := gops.Listen(gops.Options{}); err != nil {
-				fmt.Fprintf(os.Stderr, "unable to start gops: %s", err)
-				os.Exit(1)
+			addr := fmt.Sprintf("127.0.0.1:%d", viper.GetInt(option.GopsPort))
+			addrField := logrus.Fields{"address": addr}
+			if err := gops.Listen(gops.Options{Addr: addr}); err != nil {
+				log.WithError(err).WithFields(addrField).Fatal("Cannot start gops server")
+			} else {
+				log.WithFields(addrField).Info("Started gops server")
```

Comment:
Nit: `else` branch here is not necessary given the `Fatal` above.

Reply:
fixed